src/core/client/ui/components/Timestamp/Timestamp.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import TimeAgo, { Formatter } from "react-timeago";
+
+import { UIContext } from "talk-ui/components";
+import { withStyles } from "talk-ui/hocs";
+import { PropTypesOf } from "talk-ui/types";
+
+import * as styles from "./Timestamp.css";
+
+interface InnerProps {
+  date: string;
```

Comment:
We should add support for `className` here.

Reply:
👍